tests/e2e/install_test.go

```diff
@@ -143,7 +146,7 @@ var _ = Describe("E2E - Install Rancher Manager", Label("install"), func() {
 			err = kubectl.RunHelmBinaryWithCustomErr("repo", "update")
 			Expect(err).To(Not(HaveOccurred()))
 
-			err = kubectl.RunHelmBinaryWithCustomErr("install", "elemental-operator", "elemental-operator/elemental-operator",
+			err = kubectl.RunHelmBinaryWithCustomErr("upgrade", "--install", "elemental-operator", "elemental-operator/elemental-operator",
```

Comment:
This will install the elemental-operator helm chart and elemental-operator image from the github builds in any case, however it would be nice to that this is pulled form github or OBS depending on test. This turns to be slightly tricky as in OBS there is no helm chart repository published, the helm chart lives in an oci image. 

I install from OBS helm chart with
```
helm upgrade --create-namespace -n cattle-elemental-system --install elemental-operator oci://registry.opensuse.org/isv/rancher/elemental/dev/charts/elemental/elemental-operator
```

So it requires little refactor to adapt this code to make elemental-operator installation source parametrizable. In any case I believe this can be done in follow up PRs :wink:

Reply:
Yes, thx for the command line.

I would like to pair this change with this issue: https://github.com/rancher/elemental/issues/412.

Reply:
yes, makes complete sense :+1: